oemoflex/tools/plots.py

```diff
@@ -531,3 +531,46 @@ def _eng_format(ax, unit):
     formatter0 = EngFormatter(unit=unit)
     ax.yaxis.set_major_formatter(formatter0)
     return ax
+
+
+def plot_grouped_bar(ax, df, color_dict, unit, stacked=False):
+    r"""
+    This function plots scalar data as grouped bar plot. The index of the DataFrame
+    will be interpreted as groups (e.g. regions), the columns as different categories (e.g. energy
+    carriers) within the groups which will be plotted in different colors.
+
+    Parameters
+    ----------
+    ax: matplotlib Axes object
+        Axes to draw the plot.
+    df: pd.DataFrame
+        DataFrame with an index defining the groups and columns defining the bars of different color
+        with in the group.
+    color_dict: dict
+        Dictionary defining colors of the categories
+    unit: str
+        Unit of the variables
```

Comment:
For consistency reasons:

```suggestion
        Unit of the variables.
```